pkg/reader/ccip_old_sync.go

```diff
@@ -0,0 +1,128 @@
+package reader
```

Comment:
This file contains deprecated code that will be removed in a followup PR.

The code was moved from `ccip.go` and only has minor changes to function signatures.